### ISSUE TITLE
Normalize input using numpy only for onnx inference.

### ISSUE THREAD

**Issue by sagunkayastha** [lukemelas/EfficientNet-PyTorch#255]
I have retrained an efficentnet-b1 model on a custom dataset using pre-trained weights.
It works fine and gives correct result when i normalize the image using the torch transform with transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225]).

 but when i use simple numpy normalization the outputs are worng.
img -= [0.485, 0.456, 0.406]
img /= [0.229, 0.224, 0.225]

What am I doing wrong here?

Thanks in advace.

**Comment by lukemelas** [comment]
Can you give a complete reproducible example?
